### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by vinaygrp1**

### CODE REVIEW

**Comment by vinaydms** [comment]
why you add oil to karam dosa already oil is there for plain dosa specifically

**Comment by vinaygrp1** [comment]
> why you add oil to karam dosa already oil is there for plain dosa specifically
extra oil is good for karam dosa

File: menu.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-dosa
\ No newline at end of file
+dosa
+karam
+oil
```
Comment (vinaydms) [inline menu.txt:3]: why oil is added at the end ?

**Review by vinaydms — APPROVED** [review]
everything is good, we can go ahead.

**Pull request merged**